front-end/jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@beeeeesankun -san
I think no need to update this file.
Thanks

Reply:
@beeeeesankun -san
Sorry maybe I have a mistake here.
You mean config to import at the root component that right.
It's the good solution.
Thank you.